recipes/gstreamer/build.sh

```diff
@@ -8,6 +8,7 @@ cp -r $PREFIX/lib/libffi* $PREFIX/lib64
 # be included in the package (it is about 12MB).
 ./configure --disable-examples --prefix="$PREFIX" --datarootdir=`pwd`/tmpshare
```

Comment:
What do you mean?

El 26/08/16 a las 09:39, Filipe escribió:
>
> In recipes/gstreamer/build.sh 
> <https://github.com/conda-forge/staged-recipes/pull/1364#discussion_r76429555>:
>
> > @@ -8,6 +8,7 @@ cp -r $PREFIX/lib/libffi* $PREFIX/lib64
> >  # be included in the package (it is about 12MB).
> >  ./configure --disable-examples --prefix="$PREFIX" --datarootdir=`pwd`/tmpshare
>
> Why not |share|? Isn't that the default?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/conda-forge/staged-recipes/pull/1364/files/13aac897ecb7e6aaa5390fe78e508bd566b027bd..2afb4728566a0f548f06dd78d23a85c73853a589#r76429555>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AAWS7RfEt9JrBnl5ECZpzs3Ko4JmPncUks5qjvq0gaJpZM4JtMK_>.
>